src/components/Campaign.tsx

```diff
@@ -53,13 +71,37 @@ export const Campaign = () => {
           <Header>
             <EditorTopBar
               data-testid={editorTopBarTestId}
-              onSave={save}
               title={
                 campaignContentQuery.data?.campaignName
                   ? campaignContentQuery.data.campaignName
                   : ""
               }
-            />
+            >
+              <ul className="ed-header-list">
```

Comment:
This `<ul>` is duplicated, is it fine?

<img width="822" alt="image" src="https://user-images.githubusercontent.com/1157864/214550828-d547a21c-3d9c-4a0e-b559-412a786f1d84.png">

Reply:
Yes is fine 👍🏻

Reply:
Why do we need a list inside another list?

Reply:
This for more html semantic and reusability of our css, do you agree @GusBaamonde?

Reply:
The current HTML structure is something like this (pseudo-code):

```html
<div className="editor-top-bar" {...otherProps}>
  <ul className="ed-header-list">
    <li>
      <ExitDropDown />
    </li>
    <li>
      <Title />
    </li>
    <li>
      <ul className="ed-header-list">
        <li>
          <SaveAsTemplateButton>
        </li>
        <li>
          <SaveButton>
        </li>
      </ul>
    </li>
  </ul>
</div>
```

What is the semantic meaning of the two nested lists with the class _ed-header-list_?

Reply:
We have talked with @Cromeror, the last element in a `ed-header-list` list is moved to the right, so, the new button list should be inside the third element.

We agree that the buttons should be in a list, and for practical reasons, it is convenient to use the same `ed-header-list` class.

Reply:
:+1: